portfolio/src/main/java/com/google/sps/data/Image.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps.data;
+
+// Public immutable class representing an image uploaded to a website
+public class Image {
+  private final String blobKey; 
+      //the image can be accessed at "/image-handler?blob-key=${blobKey}
```

Comment:
The comment looks out of place here. Maybe add it before the blobKey declaration.

Also another nit space after '//'

Reply:
Done, thanks!